BrowserKit/Sources/Common/Constants/AppConstants.swift

```diff
@@ -10,6 +10,9 @@ public final class AppConstants {
     || AppConstants.isRunningUITests
     || AppConstants.isRunningPerfTests
 
+    public static let useAppleIntelligence =
+        !ProcessInfo.processInfo.arguments.contains(LaunchArguments.SkipAppleIntelligence)
```

Comment:
🤔 The `LaunchArgument` tends to use the pattern `Skip` something but `AppConstants` tends to say `is` something.